recipes/nysdec_state_facility_permits/create.sql

```diff
@@ -1,3 +1,44 @@
+/*
+DESCRIPTION:
+    1. Import geocoded nysdec_state_facility_permits to EDM database using PSTDIN
+    2. Create geometry from geosupport fields
+INPUTS: 
+	PSTDIN >> 
+    TEMP nysdec_state_facility_permits (
+                        facility_name text,
```

Comment:
isn't this exactly the same as the 
CREATE TEMP TABLE nysdec_state_facility_permits (
below? why do we need to document twice here?

Reply:
I would say we can remove `INPUTS` completely in all `create.sql`